src/core/CoolLog.h

```diff
@@ -26,7 +26,7 @@
     Serial.print(F(":"));                                                      \
     Serial.print(__LINE__);                                                    \
     Serial.println(F("]"));                                                    \
-    Serial.println("HEAP: ");                                                  \
+    Serial.print("HEAP: ");                                                    \
```

Comment:
remove trailing whitespaces after the `;` (install Whitespace+ extension in visual studio)